example/v1/types_stable.go

```diff
@@ -67,6 +67,18 @@ type StableConfigTypeSpec struct {
 	// celUnion demonstrates how to validate a discrminated union using CEL
 	// +optional
 	CELUnion CELUnion `json:"celUnion,omitempty"`
+
+	// backendQuotaGiB sets the etcd backend storage size limit in gibibytes.
+	// The value should be an integer not less than 8 and not more than 32.
+	// When not specified, the default value is 8.
+	// +kubebuilder:default:=8
+	// +kubebuilder:validation:Minimum=8
+	// +kubebuilder:validation:Maximum=32
+	// +kubebuilder:validation:XValidation:rule="self>=oldSelf",message="etcd backendQuotaGiB may not be decreased"
+	// +openshift:enable:FeatureGate=EtcdBackendQuota
+	// +default:=8
+	// +optional
+	BackendQuotaGiB int32 `json:"backendQuotaGiB,omitempty"`
```

Comment:
Lets do the example as a separate PR, and please make it generic, we are looking to show a specific thing here which is the defaulting and min/max, we don't need the full type. See other examples and how they're set up